dev/tools/gen_keycodes/lib/key_data.dart

```diff
@@ -133,6 +154,13 @@ class KeyData {
   /// JSON.
   Map<String, List<String>> _nameToGlfwName;
 
+  /// The mapping from the Flutter name (e.g. "eject") to the GTK name (e.g.
+  /// "GDK_KEY_Eject").
```

Comment:
GDK?
```suggestion
  /// "GTK_KEY_Eject").
```

Reply:
GDK is the low-level part of GTK (GDK = Gimp Drawing Kit, GTK = Gimp Toolkit in original naming), so events are in the GDK namespace.

Reply:
Ahh, OK, makes sense.  I was just pattern matching...